test/src/main/webapp/META-INF/context.xml-h2

```diff
@@ -1,8 +1,8 @@
-<Context>	
-	<Resource 	
-		name="jdbc/ibis4test"	
-		type="org.h2.jdbcx.JdbcDataSource"	
-		factory="org.apache.naming.factory.BeanFactory"	
-		URL="jdbc:h2:file:./ibis4test;MODE=Oracle"	
-	/>	
+<Context>
+	<Resource
+		name="jdbc/ibis4test"
+		type="org.h2.jdbcx.JdbcDataSource"
+		factory="org.apache.naming.factory.BeanFactory" 
+		URL="jdbc:h2:file:./ibis4example;MODE=Oracle;DB_CLOSE_ON_EXIT=FALSE;AUTO_SERVER=TRUE"
```

Comment:
ibis4test lijkt me een betere naam hier.
Bijvoorkeur ook niet in current directory ('.') maar in zoiets als $database.dir